samples/FileBasedMcpServer/README.md

```diff
@@ -0,0 +1,64 @@
+# File-Based MCP Server Sample
+
+This sample demonstrates how to create a complete MCP (Model Context Protocol) server using .NET 10's file-based programs feature. Unlike traditional .NET projects that require a `.csproj` file, file-based programs allow you to write and run complete applications in a single `.cs` file.
+
+## Requirements
+
+- .NET 10 SDK (RC2 or later)
+- No project file required!
+
+## Running the Sample
+
+Simply run the Program.cs file directly:
+
+```bash
+dotnet run Program.cs
+```
+
+The server will start and listen for MCP messages on stdin/stdout (stdio transport).
+
+### Making it Executable (Unix/Linux/macOS)
+
+On Unix-like systems, you can make the file executable:
```

Comment:
Again, this is documenting a feature of the .NET SDK. It's good we're pointing this out given that it's a new feature but ideally it should be pointing to .NET SDK documentation.